docs/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -540,4 +540,10 @@ Learn more about fixing these caching issues in the <PlatformLink to="/sourcemap
   </Expandable>
 </PlatformSection>
 
+<PlatformSection supported={['javascript.nextjs']}>
+  <Expandable permalink title="Using the Sentry Next SDK in a Nx monorepo">
+    If you want to set up Sentry's Next.js SDK in an Nx monorepo context, please refer to [this guide](https://nx.dev/recipes/next/next-config-setup#manually-composing-plugins-nx-15-and-prior).
```

Comment:
```suggestion
    To set up Sentry's Next.js SDK in an Nx monorepo, please refer to [this guide](https://nx.dev/recipes/next/next-config-setup#manually-composing-plugins-nx-15-and-prior).
```

The guide above is actually harmful af 🤦

I would almost put this code snippet here:

```ts
// next.config.js

const nextConfig = {
  // ...
};

const plugins = [
  // Your plugins excluding withNx
];

module.exports = async (phase, context) => {
  let updatedConfig = plugins.reduce((acc, fn) => fn(acc), nextConfig);

  // Apply the async function that `withNx` returns.
  updatedConfig = await withNx(updatedConfig)(phase, context);

  return updatedConfig;
};

// The Sentry plugin should always be applied last
const { withSentryConfig } = require('@sentry/nextjs');
module.exports = withSentryConfig(module.exports)
```